scss/components/_header.scss

```diff
@@ -0,0 +1,97 @@
+.header {
+  background: url(../../img/bg-main.jpg) no-repeat center top;
+  text-align: center;
+  background-size: cover;
+  &__head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__main {
+    padding: 20% 0;
+  }
+  &__subheading {
+    margin: 0 0 15px;
+    font-family: $secondary-font;
+    color: $primary;
+    font-size: 36px;
+    font-weight: 700;
+    font-style: italic;
+    line-height: 38px;
+    opacity: 0.7;
+  }
+  &__heading {
+    margin: 0 0 25px;
+    font-size: 100px;
+    font-weight: 700;
+    color: $primary;
+    line-height: 104px;
+    letter-spacing: 6px;
+    text-transform: uppercase;
+  }
+  &__text {
+    margin: 0;
+    font-weight: 400;
+    font-size: 14px;
+    color: $primary;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+  }
+  &__text-link {
+    margin: 0;
+    color: $primary;
+    font-weight: 700;
+    &:hover {
+      text-decoration: none;
+    }
+  }
+  @media only screen and (max-width: $md) {
+    &__subheading {
+      font-size: 30px;
+      line-height: 32px;
+    }
+    &__heading {
+      margin: 0 0 20px;
+      font-size: 80px;
+      line-height: 90px;
+      letter-spacing: 4px;
+    }
+    &__text {
+      font-size: 12px;
+    }
+  }
+  @media only screen and (max-width: $sm) {
+    &__subheading {
+      font-size: 25px;
+      line-height: 28px;
+    }
+    &__heading {
+      margin: 0 0 15px;
+      font-size: 55px;
+      line-height: 60px;
+      letter-spacing: 2px;
+    }
+    &__text {
+      font-size: 11px;
+    }
+    .logo {
+      width: 100px;
+    }
+  }
+  @media only screen and (max-width: $xs) {
+    &__subheading {
+      font-size: 20px;
+      line-height: 24px;
+    }
+    &__heading {
+      font-size: 40px;
+      line-height: 50px;
+    }
+    &__text {
+      font-size: 10px;
+    }
+    .logo {
```

Comment:
same - why are you using cascade here? Please review your code and fix similar cases